tests/cron_job_tests/test_update_ehr_status.py

```diff
@@ -166,14 +168,25 @@ def assert_ehr_data_matches(self, had_ehr_status: EhrStatus, currently_has_ehr,
         self.assertEqual(latest_ehr_time, participant_summary.ehrUpdateTime)
 
         # Check generated data
+        expected_pdr_status_str = \
+            str(EhrStatus.NOT_PRESENT) if participant_summary.ehrStatus is None else str(participant_summary.ehrStatus)
```

Comment:
Confused about how the `participant_summary` column could ever be None if EhrStatus.NOT_PRESENT is defined as the default in the model?

Reply:
I guess I had put it in as a safeguard, in case it ends up as None in some scenario now or in the future. It might not be None.